arch/cortex-m/src/scb.rs

```diff
@@ -293,3 +293,15 @@ pub unsafe fn reset() {
 pub unsafe fn set_vector_table_offset(offset: *const ()) {
     SCB.vtor.set(offset as u32);
 }
+
+/// Disable the FPU
+pub unsafe fn disable_fpca() {
+    SCB.cpacr
+        .modify(CoprocessorAccessControl::CP10::CLEAR + CoprocessorAccessControl::CP11::CLEAR);
+
+    if SCB.cpacr.read(CoprocessorAccessControl::CP10) != 0
+        || SCB.cpacr.read(CoprocessorAccessControl::CP11) != 0
+    {
+        panic!("Unable to disable FPU");
+    }
```

Comment:
Section 4.6.6 suggests that a dsb and isb instruction are needed after setting these bits: https://www.st.com/resource/en/programming_manual/dm00046982-stm32-cortex-m4-mcus-and-mpus-programming-manual-stmicroelectronics.pdf
Perhaps that would remove the need for the svc call?

Reply:
I tested with these in and I still need the svc call.